packages/babel-types/src/definitions/experimental.js

```diff
@@ -89,6 +89,32 @@ defineType("OptionalMemberExpression", {
   },
 });
 
+defineType("PipelineTopicExpression", {
+  builder: ["expression"],
+  visitor: ["expression"],
+  aliases: ["Expression"],
+  fields: {
+    expression: {
+      validate: assertNodeType("Expression"),
+    },
+  },
+});
+
+defineType("PipelineBareFunction", {
+  builder: ["callee"],
+  visitor: ["callee"],
+  aliases: ["Expression"],
```

Comment:
I am not sure that we want this to be considered an `Expression`, since it is very context-dependent.

If `path.isExpression()` is true, I would expect to be able to replace it with something without changing its behavior, for example `foo.bar` -> `_x = foo.bar`. Since this isn't true in this case, I don't think that we should alias it but I'm open to discussion.

Reply:
Noted. I'll try to change this.